doc/src/Errors_warnings.rst

```diff
@@ -734,20 +609,14 @@ Doc page with :doc:`ERROR messages <Errors_messages>`
    assumed to also be for all atoms.  Thus the pressure printed by thermo
    could be inaccurate.
 
-*The fix ave/spatial command has been replaced by the more flexible fix ave/chunk and compute chunk/atom commands -- fix ave/spatial will be removed in the summer of 2015*
-   Self-explanatory.
-
-*The minimizer does not re-orient dipoles when using fix efield*
+*The fix ave/spatial command has been replaced by the more flexible fix ave/chunk and compute chunk/atom commands -- fix ave/spatial will be removed*The minimizer does not re-orient dipoles when using fix efield*
```

Comment:
Is this supposed to be two lines?

Reply:
No, the first part should have been deleted. The warning does not exist anymore.